### DISCUSSION TITLE
14 febbraio 2023 B - esercizio 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#34] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/04aeb5d5-7635-4a46-a8b8-4bbbc6a1118e)

**Comment by M1SC0** [comment]
<img width="1123" height="910" alt="image" src="https://github.com/user-attachments/assets/ac6d6143-7bf4-4713-9532-42bea95ddd36" />
